pkg/booleanpolicy/violationmessages/printer/vuln.go

```diff
@@ -24,6 +24,12 @@ const (
     {{- if .FixedBy}}Fixable {{end}}{{.CVE}}{{if .CVSS}} (CVSS {{.CVSS}}){{end}}{{if .Severity}} (severity {{.Severity}}){{end}} found
     {{- if .Component}} in component '{{.Component}}' (version {{.ComponentVersion}}){{end}}
     {{- if .ContainerName }} in container '{{.ContainerName}}'{{end}}
+    {{- if .FixedBy}}, resolved by version {{.FixedBy}}{{end}}`
+
+	nvdcveTemplate = `
```

Comment:
If indeed two templates are needed, please add a sample message in the comments like the template above.

Reply:
@clickboo  Here is sample message from unit test
CVE-2014-0160 (NVD CVSS 8) (severity Unknown) found in component 'heartbleed' (version 1.2) in container 'nginx'